pom.xml

```diff
@@ -514,79 +430,67 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 			<artifactId>mail</artifactId>
 			<version>1.4.1</version>
 		</dependency>
-		<dependency>
-			<groupId>javax.activation</groupId>
-			<artifactId>activation</artifactId>
-			<version>1.1.1</version>
-		</dependency>
-		<dependency>
-			<groupId>javax.annotation</groupId>
-			<artifactId>javax.annotation-api</artifactId>
-			<version>1.3.1</version>
-		</dependency>
 		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>9.4.1212.jre7</version>
-		</dependency>
-		<!-- http://mvnrepository.com/artifact/net.sourceforge.jtds/jtds -->
-		<dependency>
-			<groupId>net.sourceforge.jtds</groupId>
-			<artifactId>jtds</artifactId>
-			<version>1.3.1</version>
+			<version>42.3.3</version>
 		</dependency>
 		<!-- JCHEM 16 requires dom4j 1.3. This depdendency can be removed when we upgrade jchem to jchem > 17 -->
 		<dependency>
 			<groupId>dom4j</groupId>
 			<artifactId>dom4j</artifactId>
 			<version>1.3</version>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/net.sf.trove4j/trove4j -->
 		<dependency>
-			<groupId>com.beust</groupId>
-			<artifactId>jcommander</artifactId>
-			<version>1.7</version>
+			<groupId>com.github.javadev</groupId>
+			<artifactId>underscore</artifactId>
+			<version>1.19</version>
 		</dependency>
-		<!-- https://mvnrepository.com/artifact/net.sf.trove4j/trove4j -->
 		<dependency>
```

Comment:
I see these `org.openscience.cdk` dependencies are being used by the LsThing StructureService. Could you add a comment noting that, just so we don't get confused between this and the available chemistry engines?

Reply:
Thanks, I added a few notes around the dependencies including this one about cdk.